rpc/opencdm_xdr.h

```diff
@@ -127,19 +130,39 @@ extern  rpc_response_generic * rpc_open_cdm_is_type_supported_1_svc(rpc_request_
 #define RPC_OPEN_CDM_MEDIAKEYS 2
 extern  rpc_response_generic * rpc_open_cdm_mediakeys_1(rpc_request_mediakeys *, CLIENT *);
 extern  rpc_response_generic * rpc_open_cdm_mediakeys_1_svc(rpc_request_mediakeys *, struct svc_req *);
+#if WPE
+#define RPC_OPEN_CDM_MEDIAKEYS_SET_SERVER_CERTIFICATE 3
+#define RPC_OPEN_CDM_MEDIAKEYS_CREATE_SESSION 4
+#else	//Chromium
```

Comment:
The changes you are making to RPC layer and all the #ifdef WPE mean we can't have a common CDMi which works for both Chromium and WPE browsers at the same time.

One reason for the OpenCDM architecture and having "browser glue" in the OpenCDM plugin is to abstract away the browser differences so the other parts can be browser agnostic. By putting #ifdef BROWSER in this layer we are breaking this abstraction, and moving browser specific code into the CDMi.

Reply:
Maybe, I will restructure these changes so that WPE support coexist together with existing Chrome support. Currently browser support is based on compile type option - WPE/Chrome.